labs/fabric-healthcare-payer-prescription

```diff
@@ -0,0 +1,23 @@
+# Lab Name
+Modern Pharmacy Management
```

Comment:
Lab Name should match name of markdown file. So either Lab Name is `Fabric Healthcare Payer Prescription` or name of the markdown file should be `modern-pharmacy-management.md`

Reply:
Updated as per the suggestion